JumpscaleCore/clients/tests/test_sshkey_client.py

```diff
@@ -17,15 +17,15 @@ def setUp(self):
 
     def tearDown(self):
         self.delete_from_sshdir()
-        self.delete_client_method(self.sshkey_client, "jumpscale.sshkey.client", self.SSHKEYCLIENT_NAME)
+        self.delete_client_method(self.sshkey_client, "jumpscale.sshkey.client", self.sshkeyclient_name)
 
     def delete_from_sshdir(self):
-        self.info("check delete_from_sshdir method in sshkey client")
+        self.info("Check delete_from_sshdir method in sshkey client, which delete ssh key files from ssh directory")
```

Comment:
- no need to do this method, check can be done once in targeted testcase and doesn't need to be done again 
- there is also method with the same name in base_test, what is it for?